src/UUPDownload/CLIOptions.cs

```diff
@@ -58,6 +58,9 @@ internal class DownloadRequestOptions
 
         [Option('l', "language", HelpText = @"The language to get. Must be used with the edition parameter. Omit either of these to download everything. Example: en-US", Required = false, Default = "")]
         public string Language { get; set; }
+
+        [Option('z', "releasetype", HelpText = @"The release type to report to the Windows Update servers. Example: Production, other example: Test", Required = false, Default = "Production")]
```

Comment:
We don't need more than one example, thanks though. Can you trim that help string?